test/unit/trace.test.js

```diff
@@ -429,10 +429,10 @@ tap.test('when serializing asynchronously', (t) => {
 
         t.equal(json[1], 1234)
         t.equal(trace, details.trace)
+        t.end()
```

Comment:
You don't need this.  this test is returning a promise so as long as it resolves the test should end properly